sip-core/src/main/java/de/ikor/sip/foundation/core/actuator/health/scheduler/ScheduledHealthCheck.java

```diff
@@ -1,34 +1,40 @@
 package de.ikor.sip.foundation.core.actuator.health.scheduler;
 
+import de.ikor.sip.foundation.core.actuator.health.CamelEndpointHealthMonitor;
+import de.ikor.sip.foundation.core.actuator.health.EndpointHealthIndicator;
+import java.util.Map;
+import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.actuate.health.HealthEndpoint;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
 /**
  * Scheduled execution for connection checks.
  *
- * <p>Switched off by default Default fixed delay (interval): every 15 minutes, or 900000ms Default
- * initial delay: 10 seconds, or 10000ms
+ * <p>Switched on by default. Default fixed delay (interval): every 15 minutes, or 900000ms Default
+ * initial delay: 5 seconds, or 5000ms
  *
- * <p>sip.core.metrics.scheduled-health-check.enabled:false
+ * <p>sip.core.metrics.scheduled-health-check.enabled:true
  * sip.core.metrics.scheduled-health-check.fixed-delay:900000
- * sip.core.metrics.scheduled-health-check.initial-delay:10000
+ * sip.core.metrics.scheduled-health-check.initial-delay:5000
  */
 @Service
 @Slf4j
-@ConditionalOnExpression("${sip.core.metrics.scheduled-health-check.enabled:false}")
+@AllArgsConstructor
+@ConditionalOnExpression("${sip.core.metrics.scheduled-health-check.enabled:true}")
 public class ScheduledHealthCheck {
 
-  @Autowired private HealthEndpoint healthEndpoint;
+  private final CamelEndpointHealthMonitor monitor;
 
   /** Scheduled health check */
   @Scheduled(
       fixedDelayString = "${sip.core.metrics.scheduled-health-check.fixed-delay:900000}",
-      initialDelayString = "${sip.core.metrics.scheduled-health-check.initial-delay:10000}")
+      initialDelayString = "${sip.core.metrics.scheduled-health-check.initial-delay:5000}")
   public void scheduledExecution() {
-    log.info("sip.core.health.applicationstatus_{}", healthEndpoint.health().getStatus());
+    Map<String, EndpointHealthIndicator> healthIndicators = monitor.getHealthIndicators();
+
+    healthIndicators.forEach(
```

Comment:
Looks good
This might be even better (clearer):
`    healthIndicators
            .values()
            .forEach(EndpointHealthIndicator::executeHealthCheck)`